mc2/controllers/docker/tests/test_docker_controller.py

```diff
@@ -83,7 +83,8 @@ def check_and_clear_appdata(appdata, controller):
     assert appdata.pop("cpus") == controller.marathon_cpus
     assert appdata.pop("mem") == controller.marathon_mem
     assert appdata.pop("instances") == controller.marathon_instances
-    check_and_remove_optional(appdata, "cmd", controller.marathon_cmd)
+    if controller.marathon_cmd:
+        check_and_remove_optional(appdata, "args", [controller.marathon_cmd])
```

Comment:
The reason it's needed is because `None` itself resolves to `False` which is checked in `check_and_remove_optional` but `[None]` doesn't resolve to False.

It does feel crappy though, I'll see what I can do better here.